academy/4-my-own-chain/reject-game.md

```diff
@@ -13,24 +13,24 @@ Before proceeding, make sure you have all you need:
 
 * You understand the concepts of [transactions](../2-main-concepts/transactions.md), [messages](../2-main-concepts/messages.md)), and [Protobuf](../2-main-concepts/protobuf.md).
 * You know how to [create a message](./create-message.md) with Ignite CLI, and code [its handling](./create-handling.md). This section does not aim to repeat what can be learned in earlier sections.
-* Have Go installed.
-* The checkers blockchain codebase with the previous messages and their events. You can get there by following the [previous steps](./events.md) or checking out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/two-events).
+* Go is installed.
+* You have the checkers blockchain codebase with the previous messages and their events. If not, follow the [previous steps](./events.md) or check out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/two-events).
 
 </HighlightBox>
 
 If anyone can create a game for any two other players, it is important to allow a player to reject a game. But a player should not be allowed to reject a game once they have made their first move.
```

Comment:
@xavierlepretre To confirm: "anyone can create a game for two **other** players". This implies:

- Alice could start a game for Bob and Charlie (ie _Alice creates the game but does not actually play_)

Correct?

Reply:
Yes, this is how it is implemented.